sale_invoice_group_method/model/partner.py

```diff
@@ -11,3 +11,9 @@ class ResPartner(models.Model):
         string='Default Invoice Group Method',
         comodel_name='sale.invoice.group.method'
     )
+
+    def _get_invoice_group_method_id(self):
+        if self.invoice_group_method_id:
+            return self.invoice_group_method_id
+        else:
+            return self.commercial_partner_id.invoice_group_method_id
```

Comment:
@Roodin no need to else part

Reply:
Both are needed, but a compact way to put it is:
```python
return self.invoice_group_method_id or self.commercial_partner_id.invoice_group_method_id
```

Reply:
@pedrobaeza  I mean method should be
```
def _get_invoice_group_method_id(self):
    if self.invoice_group_method_id:
        return self.invoice_group_method_id
    return self.commercial_partner_id.invoice_group_method_id
```

Reply:
And it's less good than my proposal as you can see.